src/main/webapp/dashboard/controller/Dashboard.controller.js

```diff
@@ -60,42 +75,133 @@ sap.ui.define(
-                           const cardData = data[2];
-                           let content = [];
-                           content.push({
+                           const connectorData = data[2];
+                           let connectorContent = [];
+                           let cardContent = [];
+                           connectorContent.push({
                              "Name" : "Konnektoren",
                              "Value": configs.length,
                              "Icon": url + "dashboard/images/Connector.png",
-                             "State" : statusConnectors,
-                             "Info": infoConnectors
+                             "State" : inactiveConnectors > 0 ? "Error" : "Success",
+                             "Info": inactiveConnectors > 0 ? "Offline: " + inactiveConnectors : "Alle Online"
                            });
-                           content.push({
+                           connectorContent.push({
                              "Name" : "Kartenterminals",
                              "Value" : numTerminals,
                              "Icon": url + "dashboard/images/CardTerminal.png",
-                             "State": statusTerminals,
-                             "Info" : infoTerminals
+                             "State": inactiveTerminals > 0 ? "Error" : "Success",
+                             "Info" : inactiveTerminals > 0 ? "Offline: " + inactiveTerminals : "Alle Online"
                            });
-                           content.push({
+                           connectorContent.push({
                              "Name" : "Karten",
                              "Value": numCards,
                              "Icon": url + "dashboard/images/Card.png",
                              "State": "None"
                            });
-                           cardData["sap.card"].content.data.json = content;
-                           Card.setManifest(cardData);
+                           cardContent.push({
+                             "cardType": "EGK",
+                             "Value": cardTypes["EGK"],
+                             "pinType": "PIN.CH",
+                             "pinVerified" : cardTypes["EGK"] > 0 ? stats["VERIFIED"]["EGK"] : "-",
+                             "pinVerifiable" : cardTypes["EGK"] > 0 ? stats["VERIFIABLE"]["EGK"] : "-",
+                             "pinTransport" : cardTypes["EGK"] > 0 ? stats["TRANSPORT_PIN"]["EGK"] : "-",
+                             "pinBlocked" : cardTypes["EGK"] > 0 ? stats["BLOCKED"]["EGK"] : "-",
+                             "pinDisabled" : cardTypes["EGK"] > 0 ? stats["DISABLED"]["EGK"] : "-",
+                             "pinEmpty" : cardTypes["EGK"] > 0 ? stats["EMPTY_PIN"]["EGK"] : "-",
+                             "validity" : cardTypes["EGK"] > 0 ? invalidCertCards["EGK"] > 0 ? "Abgelaufen: " + invalidCertCards["EGK"] : "Alle gültig" : "-",
```

Comment:
Please add "Abgelaufen: " and "Alle gültig" to the i18n file and reference them from there as you did on the Master.controller.js.